src/bookshelf/Makefile

```diff
@@ -28,7 +28,7 @@ include Makefile.db
 
 ## TODO: put 'ct' back in here
 all: $(REBAR3) elvis
-	@$(REBAR3) do clean, compile, eunit, dialyzer, xref
+	@$(REBAR3) do clean, compile, eunit, xref #, dialyzer, xref
```

Comment:
Why are we disabling dialyzer and warnings_as_errors below?

We should at least file an issue to fix whatever the problem is and turn dialyzer and warnings_as_errors back on

Reply:
yes absolutely - this was to test if there are any other unit test failures other than dialyzer.

Reply:
@markan - warnings, dialyzer, linting et al. have now been re-enabled.  In the words of Obi-wan, "warnings and linting are our specialty." :)